pagerduty_integration.go

```diff
@@ -63,6 +63,39 @@ func (c *Client) GetPagerDutyIntegration(ctx context.Context, id string) (*integ
 	return &finalIntegration, err
 }
 
+// GetPagerDutyIntegrationByName retrieves a PagerDuty integration by name.
+func (c *Client) GetPagerDutyIntegrationByName(ctx context.Context, name string) (*integration.PagerDutyIntegration, error) {
+	resp, err := c.doRequest(ctx, "GET", IntegrationAPIURL, nil, nil)
```

Comment:
This will get all integrations, not just PagerDuty ones.  

The best thing to do is to get the URL `IntegrationAPIURL + "?type=PagerDuty&name=" + name` and then iterate those results looking for the specific name (the `name` query parameter searches for a substring of the name so it is possible that you will have multiple results).

Reply:
Sure of course. I missed the query parameters there. thanks a lot

Reply:
Fixed.